Foundation/NSAttributedString.swift

```diff
@@ -186,11 +186,11 @@ private extension AttributedString {
     }
 }
 
-extension AttributedString: _CFBridgable {
+extension NSAttributedString: _CFBridgable {
```

Comment:
I was under the impression that the style tended to be
```swift
TypeName : OtherType {
    ...
}
```

for type declarations and `var name: Type = ...` for variable declarations, but I could be out of date. Across all of our code, this seems to be the most consistent styling, and AFAIR, I think this is how sample Swift code is styled, no? (Again, could certainly be wrong here.)